tsconfig.base.json

```diff
@@ -15,14 +15,17 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "@elewa-website/elements/layout/header": [
-        "libs/elements/layout/header/src/index.ts"
-      ],
       "@elewa-website/cards": ["libs/cards/src/index.ts"],
-      "@elewa-website/elements/cards": ["libs/elements/cards/src/index.ts"],
       "@elewa-website/data/ui/content-text": [
         "libs/data/ui/content-text/src/index.ts"
       ],
+      "@elewa-website/elements/cards": ["libs/elements/cards/src/index.ts"],
```

Comment:
This removes the aliased path to these modules.

Reply:
I have made all the requested changes.kindly.
@LemmyMwaura